crates/gitbutler-tauri/src/open.rs

```diff
@@ -0,0 +1,124 @@
+use crate::error::Error;
+use anyhow::anyhow;
+use regex::Regex;
+use tracing::instrument;
+
+// fix_env_variables
+use std::ffi::{OsStr, OsString};
+use std::io;
+use std::os::unix::prelude::OsStrExt;
```

Comment:
This will probably prevent it from compiling on Windows. There is [`gix::path::os_string_into_bstring()`](https://docs.rs/gix-path/latest/gix_path/fn.os_string_into_bstring.html) which is cross-platform.

But it turns out that it's probably better to work with `PathBuf` here.